includes/class-wc-subscriptions-core-plugin.php

```diff
@@ -286,15 +286,6 @@ public function get_product_type_name() {
 		return 'subscription';
 	}
 
-	/**
-	 * Gets the activation transient name.
-	 *
-	 * @return string The transient name used to record when the plugin was activated.
-	 */
-	public function get_activation_transient() {
```

Comment:
Let's keep this `get_activation_transient()` functionality in WCS Core. While we're not using in core at the moment, this function could be helpful down the track so it's best to avoid moving it now and then having to move it back or duplicate this function.

Same goes for setting the transient below:

```
set_transient( $this->get_activation_transient(), true, 60 * 60 );
```

Reply:
(I'll leave the decision in your hands @mattallan - wanted to share my rationale for the alternative.)